torch_geometric/utils/isolated.py

```diff
@@ -37,7 +37,10 @@ def contains_isolated_nodes(
     return torch.unique(edge_index.view(-1)).numel() < num_nodes
 
 
-def remove_isolated_nodes(edge_index, edge_attr=None, num_nodes=None):
+def remove_isolated_nodes(
+    edge_index: Tensor, edge_attr: Optional[Tensor] = None,
+    num_nodes: Optional[int] = None
```

Comment:
```suggestion
    num_nodes: Optional[int] = None,
```
and let GitHub auto-format.